rust/element/src/data.rs

```diff
@@ -113,6 +120,13 @@ pub enum Syntax<'a> {
     /// Greater element
     CenterBlock,
 
+
+
+
+
+
+
+
```

Comment:
empty lines will be removed as long as #46 is merged first, no changes needed in this commit

Reply:
#46 is merged. The empty lines did not disappear.